Integrations/integration-Guidance_Encase_Endpoint.yml

```diff
@@ -20,7 +20,7 @@ configuration:
   name: port
   defaultvalue: ""
   type: 0
-  required: true
+  required: false
```

Comment:
We will leave it required.
We didn't found the original field that was not required.